cmake/msan-blacklist.txt

```diff
@@ -0,0 +1 @@
+fun:longest_match
```

Comment:
This is not mentioned in commit message?

Reply:
I mentioned it now (disabling a false positive in zlib)